.github/workflows/tests.yml

```diff
@@ -56,7 +53,7 @@ jobs:
 
             - name: "Use PHPUnit 9.3+ on PHP 8.0 & PHP 8.1"
               run: composer require --no-update --dev phpunit/phpunit:^9.3
-              if: "matrix.php == '8.0' || matrix.php == '8.1'"
+              if: "matrix.php == '8.0' || matrix.php == '8.1' || matrix.php == '8.2' || matrix.php == '8.3' || matrix.php == '8.4'"
```

Comment:
I'm sure there's a better way to do this, but I'm authoring this from my phone while on vacation, so I'm just going to copy-paste this